### PR TITLE
Fix install/ version update scripts

### PR DESCRIPTION
**Pull request by joestringer**
Due to a misunderstanding of Makefile `$(shell ...)`, the versions in the Makefile were not being respected.

This PR fixes up the Makefile and bumps it to reflect the versions currently in the tree.

Also, bump hubble-ui to 0.7.3 ( intended for the upcoming v1.9.0 cilium release).

### CODE REVIEW

File: install/kubernetes/cilium/values.yaml [hunk 1]
```diff
@@ -428,7 +428,7 @@ certgen:
   image:
     repository: docker.io/cilium/certgen
     tag: v0.1.3@sha256:460fdcbabfa08b435dbee455e88add1ec54761ca2693e1d9885408ff55ed7f8e
-    pullPolicy: IfNotPresent
+    pullPolicy: Always
```
Comment (sayboras) [inline install/kubernetes/cilium/values.yaml:431]: nit: version + sha256 combination seems unique to me, can you explain why this got changed to always ?

File: install/kubernetes/Makefile [hunk 2]
```diff
@@ -7,10 +7,11 @@ include ../../Makefile.defs
 VERSION := 1.8.90
```
Comment (sayboras) [inline install/kubernetes/Makefile:7]: not related to this PR, I just saw this line, this can be addressed separately.

**Review by genbit — APPROVED** [review]

**Review by sayboras — APPROVED** [review]
LGTM, one minor nit.

**Review by genbit — APPROVED** [review]

File: install/kubernetes/cilium/values.yaml (hunk 1 above)
Comment (joestringer) [reply install/kubernetes/cilium/values.yaml:431]: `Always` is the pullPolicy by default on the latest version of the tree, we only change it to `IfNotPresent` when we create branch versions or release versions.

File: install/kubernetes/Makefile (hunk 2 above)
Comment (joestringer) [reply install/kubernetes/Makefile:7]: https://github.com/cilium/cilium/pull/13863

**Pull request merged**